internal/providers/azure/pke/adapter/gorm_store.go

```diff
@@ -354,7 +357,7 @@ func (s gormAzurePKEClusterStore) Create(params pke.CreateParams) (c pke.PKEOnAz
 				DesiredMem:          params.ScaleOptions.DesiredMem,
 				DesiredGpu:          params.ScaleOptions.DesiredGpu,
 				OnDemandPct:         params.ScaleOptions.OnDemandPct,
-				Excludes:            marshalStringSlice(params.ScaleOptions.Excludes),
+				Excludes:            excludes.(string),
```

Comment:
is this safe here?

Reply:
I added a more robust conversion to be sure.